Lesson32/Merve/src/components/Exercise4.js

```diff
@@ -0,0 +1,28 @@
+const Exercise4=(props)=>{
+  const {item}= props;
+    return(
+
+        <li
+        onClick={()=>alert(item)}
+        >{item}</li>
+
+    );
+
+};
+
+export default Exercise4;
```

Comment:
There are two exports in this file that export two different components. In this case, it's good to use named exports instead of default exports.

Reply:
I didnt do anything on Exc.4 this was on the original file but i did remove it on my workspace